test/test_web.rb

```diff
@@ -141,12 +141,12 @@ def perform(a, b)
     end
 
     get "/queues/default?count=3" # direction is 'desc' by default
-    assert_match(/1005/, last_response.body)
-    refute_match(/1002/, last_response.body)
+    assert_match(/\b1005\b/, last_response.body)
+    refute_match(/\b1002\b/, last_response.body)
 
     get "/queues/default?count=3&direction=asc"
-    assert_match(/1000/, last_response.body)
-    refute_match(/1003/, last_response.body)
+    assert_match(/\b1000\b/, last_response.body)
+    refute_match(/\b1003\b/, last_response.body)
```

Comment:
This was failing because "1003" showed up in a timestamp on the page:

<img width="1286" alt="CleanShot 2022-08-20 at 09 34 58@2x" src="https://user-images.githubusercontent.com/372/185748627-3744d246-88ee-47b5-897d-18be748669ad.png">